tasks/task723_DSTC2_classification.json

```diff
@@ -18,14 +19,14 @@
         {
             "input": "0.noise\n1.cheap restaurant in the east part of town\n2.phone number\n3.noise\n4.phone number\n5.good bye\n",
             "output": "You want to find a cheap restaurant and it should be in the east part of town. You want to know the phone number.",
-            "explanation": "All the enquires of the customer with every detail is reflected"
+            "explanation": "All the enquires of the customer with every detail is reflected."
         }
     ],
     "Negative Examples": [
         {
             "input": "0.traditional food\n1.traditional\n2.spanish food\n3.spanish\n4.food\n5.address\n6.price range\n7.thank you good bye\n",
             "output": "You want to find and it should serve african food. You don't care about the price range. Make sure you get the address, phone number, and area of the venue.",
-            "explanation": "The food type did not match and the price range was mis calculated."
+            "explanation": "The food type did not match and the price range was miscalculated."
         },
         {
             "input": "0.im looking for something in the west side\n1.doesnt matter as long as it is moderately priced\n2.can i have the address of it\n3.whats its phone number\n4.thank you\n5.goodbye\n",
```

Comment:
Maybe change the explanation of this example(second negative example) to: `The defined goals do not match the requirements of the customer.`

Reply:
The distribution of the data in both these datasets are skewed in nature. The overall distribution of task 721 falls highly under class 0. It's like 180:20 ( 0 : other classes) kind of distribution. For 724, there is the same thing but I will try to make it a little less skewed.